README.md

```diff
@@ -39,66 +41,80 @@
 
 #### Built With
 
 * [Node](https://nodejs.org/en/)
-* [Nunjucks](https://mozilla.github.io/nunjucks/)
 * [MongoDB](https://www.mongodb.com/)
+* [Nunjucks](https://mozilla.github.io/nunjucks/)
 
-<!-- GETTING STARTED -->
 ## Getting Started
 
-To get started with ApostropheCMS, follow these steps to set up a local development environment.
+To get started with ApostropheCMS, follow these steps to set up a local development environment. For more detail, refer to the [Getting Started tutorial](https://docs.apostrophecms.org/getting-started/creating-your-first-project.html) in the documentation.
 
 #### Prerequisites
 
 We recommend installing the following with [Homebrew](https://brew.sh/) on macOS. If you're on Linux, you should use your package manager (apt or yum). If you're on Windows, we recommend the Windows Subsystem for Linux.
 
 | Software | Minimum Version | Notes
 | ------------- | ------------- | -----
-| Git  | Any
-| Xcode  | Current | Only needed on macOS
 | Node.js | 8.x | Or better
 | npm  | 6.x  | Or better
 | MongoDB  | 3.6  | Or better
 | Imagemagick (optional)  | Any | GIF support, faster image uploads
 
 
-#### New Project
-Our recomended way to start a new project with ApostropheCMS is to use the [Apostrophe CLI](https://github.com/apostrophecms/apostrophe-cli). Alternatively, you can simply clone this repo and `npm install`.
+#### Installation
+Our recomended way to start a new project with ApostropheCMS is to use the [Apostrophe CLI](https://github.com/apostrophecms/apostrophe-cli).
+
 ##### Install the Apostrophe CLI
  ```sh
   npm install -g apostrophe-cli
  ```
 ##### Create a project
- ```sh
- apos create-project <shortname-without-spaces>
- ```
-
-`<shortname-without-spaces>` represents the "shortname" of the application. It will be used by default as a MongoDB database name and a basis for cookie names, etc. You will find this as the shortName property in app.js.
-
+```sh
+apos create-project wonderful-project
+```
+
+##### Install dependencies
+```sh
+cd wonderful-project && npm install
+```
+##### Create an admin user add them to the admin group
+```sh
+node app.js apostrophe-users:add myUser admin
+```
+##### Run
+```sh
+node app.js
+```
+
+Navigate to `localhost:3000/login` to login with the admin credentials you created.
+#### Next Steps
+Check out our [Getting Started tutorial](https://docs.apostrophecms.org/getting-started/creating-your-first-project.html#working-with-areas) in the documentation to start adding content. Be sure to reference our [glossary of terms](https://docs.apostrophecms.org/reference/glossary.html) to get acquainted with the reference materials.
```

Comment:
This helps. Maybe mention "user-editable content." Because it pangs me that they get to this point and they don't yet see why Apostrophe is not Strapi.

Reply:
There is a demo at the very top of the page. To this end though, I would like to put a nice image...somewhere.

Reply:
True in A3, but there is no demo in A2. This is a PR on A2 at the moment, but I get how that soon will not matter and we'll be doing something similar in the A3 README.

Reply:
True in A3, but there is no demo in A2. This is a PR on A2 at the moment, but I get how that soon will not matter and we'll be doing something similar in the A3 README.

Reply:
The demo in the header links to open-museum.

Reply:
OK I see your point. They can't do anything interesting right away after taking the steps in the README, but they have been given a pointer to a proper demo already by that point, so they know it's possible, and they have been linked to the docs to build more.